### PR TITLE
Tried to Update ReadMe

### PR DESCRIPTION
**Pull request by Yucked**
I tried 😩

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,27 +1,31 @@
...
     WebSocketPort = 80,
     Authorization = "YOUR_SECRET_AUTHORIZATION_KEY",
-    TotalShards = 1 // Please set this to the total amount of shards your bot uses
+    TotalShards = 1 // Set this to the total amount of shards your bot uses
 });
 ```
+*Notes:* 
+> You don't have to pass a `LavalinkManagerConfig` since Sharplink uses the default config.
+
+> If you don't know what the `TotalShards` are for your bot, use `DiscordRestClient#GetRecommendedShardCountAsync()` or just set it to `1` if your bot is in <1k guilds.
```
Comment (Devoxin) [inline README.md]: Limit is ~2.5K guilds for per shard/non-sharded bots

File: README.md [hunk 2]
```diff
@@ -1,27 +1,31 @@
...
-```csharp
-DiscordSocketClient client = new DiscordSocketClient();
+- Follow [these instructions](https://github.com/Frederikam/Lavalink#server-configuration) to setup Lavalink.
+- Once Lavalink is up and running follow the code example below to setup Sharplink.
 
-// You must pass the socket client over to the manager for reference
-LavalinkManager lavalinkManager = new LavalinkManager(client, new LavalinkManagerConfig()
+```cs
+var client = new DiscordSocketClient();
+var lavalinkManager = new LavalinkManager(client, new LavalinkManagerConfig
```
Comment (Devoxin) [inline README.md]: Any reason for var and not defining type explicitly?

Comment (NovusTheory) [reply README.md]: I'll need to agree with this and that I prefer using explicit type definitions

**Comment by Yucked** [comment]
Defining types explicitly seems more of a Java style than C#.

People in Discord.Net channel suggest to begin sharding when your bot is in 1k+ guilds that's why I put < 1k.

**Pull request merged**